lib/recipe/bloc/recipe_bloc.dart

```diff
@@ -288,7 +393,9 @@ class RecipeBloc extends Bloc<RecipeEvent, RecipeState> {
     List<Map<String, dynamic>> probeOptions =
         List.from(probes[changedProbeIndex].probeOptions);
 
-    probeOptions.removeAt(event.probeOptionIndex);
+    if (probeOptions.length > 2) {
+      probeOptions.removeAt(event.probeOptionIndex);
+    }
```

Comment:
Again, I think it'd be nicer not to silently not remove a Probe without any indication in the BLoC.